scripts/install_third_party.sh

```diff
@@ -116,3 +116,39 @@ if [ ! -d "$NODE_MODULE_DIR/jscs" ]; then
   echo installing node-jscs
   $NPM_INSTALL jscs@2.3.0
 fi
+
+echo Checking whether gulp is installed
+if [ ! -d "$NODE_MODULE_DIR/gulp" ]; then
+  echo installing gulp
+  $NPM_INSTALL gulp@3.9.0
+fi
+
+echo Checking whether oppia-gulp is installed
```

Comment:
Oh, cool, you made an npm thing!

Though I was wondering, do we need it? I'd rather keep the oppia-specific stuff within the codebase...